apps/agile-project-fe/src/app/components/input/input.tsx

```diff
@@ -1,17 +1,33 @@
 import styled from 'styled-components';
 
 export interface InputProps {
-  variant: 'email';
+  variant: 'text' | 'email' | 'password';
 }
 
-const StyledInput = styled.div`
-  color: pink;
-`;
+const StyledInput = styled.div``;
+
+export function Input({ variant }: InputProps) {
```

Comment:
Please refactor this component to have the same `React.FC` type as the button component